VideoIndexerEnabledByArc/single-node-k8s-cluster/single-node.k8s.bicep

```diff
@@ -0,0 +1,215 @@
+@description('Azure Resource Name Prefux')
+param prefix string
+
+@description('Specifies control plane node VM size.')
+param controlPlaneNodeVmSize string = 'Standard_DS32a_v4'
```

Comment:
is one DS32 enough to index? did you try to upload a movie?

Reply:
not yes, changed to D64.